src/content/posts/Angor vs. Traditional Crowdfunding/index.md

```diff
@@ -0,0 +1,87 @@
+
+In this article, we compare Angor to traditional crowdfunding services, highlighting the key differences, advantages, and potential challenges.
+
+## Introduction to Angor Testing
+
+### 1. Platform Structure
+
+#### Traditional Crowdfunding Platforms:
+Centralized: Platforms like Kickstarter and Indiegogo operate as centralized entities, managing all transactions and interactions between creators and backers.
+
+Middleman Dependency: These platforms act as intermediaries, which means they have control over the fund distribution and project approval processes.
+
+#### Angor:
+Decentralized: Angor operates on a decentralized model using Bitcoin for financial transactions and Nostr for communication. This removes the need for a central authority, giving more control to users.
+
+Peer-to-Peer: The decentralized nature allows direct interactions between project founders and investors without relying on a middleman.
+
+### 2. Security and Fund Control
+
+#### Traditional Crowdfunding Platforms:
+Fund Management: In traditional platforms, the platform itself holds the funds until the project meets its goal. There is a risk associated with platform security breaches or mismanagement of funds.
+
+All-or-Nothing: Many platforms follow an all-or-nothing funding model, where funds are only released if the campaign reaches its goal. If not, funds are returned to backers.
+
+#### Angor:
+Time-Lock Contracts: Angor uses Bitcoin’s time-lock contracts, releasing funds in stages based on project milestones. This ensures funds are only accessible when specific conditions are met.
+
+Investor Control: Investors have the ability to recover unspent funds at any point, significantly reducing the risk of losing their investment due to project failure or fraud.
+
+### 3. Transparency and Accountability
+
+#### Traditional Crowdfunding Platforms:
+Limited Updates: While traditional platforms allow project updates, these are often controlled by the platform and can be infrequent or lack detail.
+
+Trust Issues: Backers must trust the platform and the project creators to be transparent and honest about progress and challenges.
+
+#### Angor:
+Nostr Integration: Angor uses Nostr, a decentralized communication protocol, for project updates. This ensures that updates are transparent and directly accessible to investors.
+
+Direct Communication: Project founders can post updates directly, maintaining accountability and fostering trust through regular, detailed communication.
+
+### 4. Costs and Fees
+
+#### Traditional Crowdfunding Platforms:
+Platform Fees: Traditional platforms charge a percentage of the funds raised, typically ranging from 5% to 10%, in addition to payment processing fees.
```

Comment:
5% to 15% actually

Reply:
- I'd keep 5-10% as a 'typical range'
- Would add the fact they are sometimes charging other fees as well (set up fees, transfer fees etc) -> goes back to the previous point and fee structure being more simple